fastestimator/network/network.py

```diff
@@ -32,7 +34,7 @@ def __init__(self, ops):
         self.current_epoch_model = {}
         self.model = {}
         self.all_losses = []
-        self.loss_list = []
+        self.losses_epoch = []
```

Comment:
this name might make more sense as epoch_losses to be more consistent with 'all_losses' from above

Reply:
fixed